Week2/js-exercises/myFavoriteHobbies/myFavoriteHobbies.js

```diff
@@ -0,0 +1,42 @@
+
+
+const myHobbies = [
+    "Meditation",
+    "Reading",
+    "Programming",
+    "Hanging out with friends",
+    "Going to the gym"
+];
+
+
+//made WITHOUT using array.map()
+
+/*function convertItemToLi(array) {
+  let unorderedList = document.createElement("ul");
+  for (let item of array) {
+    let listItem = document.createElement("li");
+    let listArr = [];
+    listItem.innerHTML = item;
+    unorderedList.appendChild(listItem);
+  }
+  console.log(unorderedList);
+  document.body.appendChild(unorderedList);
+}*/
+
+//console.log(convertItemToLi(myHobbies));
+
+
+//made WITH array.map()
+
+function convertItemsToLi(array) {
+    let unorderedList = document.createElement("UL");
+
+    array.map(item => {
```

Comment:
since you don't use the return value of map, the right array method here is .forEach